tests/test_hub_features.py

```diff
@@ -35,3 +35,25 @@ def test_subfolder(self):
             model = PeftModel.from_pretrained(model, model_id, subfolder=subfolder)
 
             assert isinstance(model, PeftModel)
+
+
+class TestLocalModel:
+    def test_local_model_saving_no_warning(self, recwarn, tmp_path):
+        # When the model is saved, the library checks for vocab changes by
+        # examining `config.json` in the model path.
+        # However, previously, those checks only covered huggingface hub models.
+        # This test makes sure that the local `config.json` is checked as well.
+        # If `save_pretrained` could not find the file, it will issue a warning. 
+        model_id = "facebook/opt-125m"
+        model = AutoModelForCausalLM.from_pretrained(model_id)
+        local_dir = tmp_path / model_id
+        model.save_pretrained(local_dir)
+        del model
+
+        base_model = AutoModelForCausalLM.from_pretrained(local_dir)
+        peft_config = LoraConfig()
+        peft_model = get_peft_model(base_model, peft_config)
```

Comment:
`LoraConfig` and `get_peft_model` need to be imported.

Reply:
Resolved.